core/store/src/test/java/uk/gov/gchq/gaffer/store/StoreTest.java

```diff
@@ -1150,6 +1152,11 @@ protected void addAdditionalOperationHandlers() {
             return addElementsHandler;
         }
 
+        @Override
+        protected OutputOperationHandler<GetTraits, Set<StoreTrait>> getGetTraitsHandler() {
+            return getTraitsHandler;
```

Comment:
This should maybe return an actual handler to retain the same functionality as `getTraits()`